CMakeLists.txt

```diff
@@ -51,6 +51,8 @@ elseif(UNIX)
         set(ULTRASCHALL_TARGET_SYSTEM "linux")
     elseif(${UNAME_OUTPUT} MATCHES "[U|u]buntu")
         set(ULTRASCHALL_TARGET_SYSTEM "linux")
+    elseif(${UNAME_OUTPUT} MATCHES "Linux")
+        set(ULTRASCHALL_TARGET_SYSTEM "linux")
```

Comment:
The platform string on my machine is:

```
Linux taban 5.11.20-300.fc34.x86_64 #1 SMP Wed May 12 12:45:10 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

The relevant bit seems to be `GNU/Linux` so I matched against `Linux` which should be safe.